src/_prebid.jsx

```diff
@@ -47,10 +62,15 @@ export default class PrebidContainer extends Component {
   }
 
   configure() {
-    const { config } = this.props;
-    if (config.pricing) {
-      pbjs.setPriceGranularity({ buckets: config.pricing });
-    }
+    const { bidderTimeout, currency, debug, priceGranularity } = this.props;
+    // The pbjs.setConfig param object must be JSON
```

Comment:
All those props area already validated by propTypes, why are u dumping and parsing it?

Reply:
They must be JSON, as stated in the docs, `undefined` is not a valid JSON value, the lib uses `for` to iterate the config properties and calling some functions, passing `undefined` it throws errors

Reply:
damm... I would rather use a lodash function to do this cleanup